src/studio/src/designer/backend/Languages/ini/en.ini

```diff
@@ -114,7 +115,9 @@ service_name_is_too_long = App name must contain at most 30 characters
 service_owner_description = The app will be visible from the owners dashboard.
 service_saved_name_description = Used (with owner) as an unique identifier for the app. Also used as name of repository, in all URLs and in the Altinn APIs. Should therefore be both short and descriptive, such as "sick-leave" or "taxform-2019".
 sorte_services = Sort
+star = Make favorite
```

Comment:
If we're following through on the english spelling. `organisation` over `organization` ect. then it should be `favourite` not `favorite `

Reply:
Right, fixed. Also removed some texts no longer used.